File: Data Structures & Algorithms/largest-unique-number/submission-1.py
class Solution:
    def largestUniqueNumber(self, nums: List[int]) -> int:
        count = {}

        for num in nums:
            if num in count:
                count[num] += 1
            else:
                count[num] = 1


        largest = -1
        for i in count:
            if count[i] == 1:
                largest = max(largest, i)
        return largest

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int currentMax = 0;
        int tempMax = 0;
        for(int num : nums) {
            if(num == 1) {
                tempMax++;
                currentMax = max(tempMax, currentMax);
            }
            else {
                tempMax = 0;
            }
        }
        return currentMax;
    }
};